mathics/core/parser/parser.py

```diff
@@ -49,7 +49,7 @@
 # FIXME: should rework so we don't have to do this
 # We have the character name ImaginaryI and ImaginaryJ, but we should
 # have the *operator* name, "I".
-special_symbols["\uF74F"] = special_symbols["\uF74E"] = "I"
+special_symbols["\uf74f"] = special_symbols["\uf74e"] = "I"
```

Comment:
This one is weird in that black or some formatter decided to do this. 

Again, I'd rather switch than argue with whatever formatter decided this.